rpm/sailfish-components-webview.spec

```diff
@@ -135,6 +136,8 @@ Unit tests and functional tests of Sailfish OS WebView / WebEngine
 
 %prep
 %setup -q -n %{name}-%{version}
+sed -i "s/^Version: .*/Version: "$(echo %{version} | sed 's/+.*//')"/" \
+    ./lib/pkgconfig/sailfishwebengine.pc
 
 %build
 %qmake5 VERSION=`echo %{version} | sed 's/+.*//'`
```

Comment:
So with above please just remove sed'ing from this line.

Reply:
This means I can close this because the version is set elsewhere? No idea why I oversaw that.

Reply:
You can use this PR. Just drop sed from line 139-140 and this line. Then pc file matches to the lib regardless of the way how this was build.

Reply:
And edit a bit the commit title to better reflect the intention. Something like "Keep pc file version and library version in sync"

Reply:
Like this?

Reply:
Yes. And now just drop your sed'ing from line 139-140 as well.